redesign.md

```diff
@@ -56,6 +56,33 @@ book to read.
 * Emphasis: ?
 * Status: Early draft released to backers of the kickstarter campaign
 
+## Web Development in Perl 6
+
+* Author: Gabor Szabo
+* Formats: ?
+* Target Audience: 
+* Emphasis: ?
+* Status: Crowdfunding
+
+## Using Perl 6
+
```

Comment:
@ash are you aware that there's already a (somewhat defunct) book called "Using Perl 6"? https://github.com/perl6/book (not saying it's wrong to reuse the name, just that it might cause confusion).